application/client.core/src/app/environment/components/sidebar/search.manager/controls/component.ts

```diff
@@ -4,13 +4,15 @@ import { ControllerSessionTabSearchStore } from '../../../../controller/session/
 import { DialogsRecentFitlersActionComponent } from '../../../dialogs/recentfilter/component';
 import { NotificationsService } from '../../../../services.injectable/injectable.service.notifications';
 import { Session } from '../../../../controller/session/session';
+import { IPCMessages } from '../../../../services/service.electron.ipc';
 
 import * as Toolkit from 'chipmunk.client.toolkit';
 
 import TabsSessionsService from '../../../../services/service.sessions.tabs';
 import HotkeysService from '../../../../services/service.hotkeys';
 import PopupsService from '../../../../services/standalone/service.popups';
 import EventsSessionService from '../../../../services/standalone/service.events.session';
+import ServiceElectronIpc from '../../../../services/service.electron.ipc';
```

Comment:
Using components to listen IPC messages isn't good idea. Please use controllers/service to handle income IPC or sending IPC message to BE